src/common/meta/src/sequence.rs

```diff
@@ -26,31 +26,67 @@ pub type SequenceRef = Arc<Sequence>;
 
 pub(crate) const SEQ_PREFIX: &str = "__meta_seq";
 
-pub struct Sequence {
-    inner: Mutex<Inner>,
+pub struct SequenceBuilder {
+    name: String,
+    initial: u64,
+    step: u64,
+    generator: KvBackendRef,
+    max: u64,
 }
 
-impl Sequence {
-    pub fn new(name: impl AsRef<str>, initial: u64, step: u64, generator: KvBackendRef) -> Self {
-        let name = format!("{}-{}", SEQ_PREFIX, name.as_ref());
-        let step = step.max(1);
+impl SequenceBuilder {
+    pub fn new(name: impl AsRef<str>, generator: KvBackendRef) -> Self {
         Self {
+            name: format!("{}-{}", SEQ_PREFIX, name.as_ref()),
+            initial: 1024,
+            step: 10,
```

Comment:
I think the default values for "initial" and "step" should be 0 and 1.
When functioning as a table ID generator, it's crucial to proactively set 'initial' to 1024 and 'step' to 10.